lib/dicon/secure_shell.ex

```diff
@@ -123,6 +123,13 @@ defmodule Dicon.SecureShell do
     format_if_error(result)
   end
 
+  def write_progress_bar(percent) do
```

Comment:
Maybe, we can add guards to make this function stricter, something like:

```elixir
def write_progress_bar(percent) when is_integer(percent) and percent in 0..100
```

Wdyt? It's also more easy to see that `percent` is an integer in `1..100` instead of a float in `[0, 1]` (even if the name `percent` already suggests that).